src/authInfo.ts

```diff
@@ -1015,19 +1028,15 @@ export class AuthInfo extends AsyncCreatable<AuthInfo.Options> {
     }
 
     // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
-    // @ts-ignore TODO: need better typings for jsforce
-    const { orgId, userId } = parseIdUrl(authFieldsBuilder.id);
+    // @ts-ignore
+    const { orgId } = parseIdUrl(authFieldsBuilder.id);
```

Comment:
userId is no longer needed.

Reply:
what's the ignore for?

Reply:
@WillieRuemmele w/o ignore I get this TS error
`TS2339: Property 'instance_url' does not exist on type 'TokenResponse'.`

I see this comment at the top of this file
```
// No typings directly available for jsforce/lib/transport
// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
// @ts-ignore
import * as Transport from 'jsforce/lib/transport';
```